app/models/school_can_order_devices_notifications.rb

```diff
@@ -48,6 +48,8 @@ def what_message_to_send(school, user)
       :user_can_order_but_action_needed
     elsif status?('rb_can_order', 'school_can_order', school: school) && user.orders_devices? && !user.seen_privacy_notice?
       :nudge_user_to_read_privacy_policy
+    elsif status?('rb_can_order', school: school) && school.responsible_body.has_virtual_cap_feature_flags? && user.in?(school.order_users_with_active_techsource_accounts)
```

Comment:
Is a preorder status of rb_can_order enough to qualify that it is 1) centrally managed, and 2) ready to order - even after placing an initial order?

Reply:
Yes i think that is sufficient. It is derived from `PreorderInformation#infer_status`. 
- It checks the `PreorderInformation` on who is ordering
- Even after an order if they can still order it looks like it will stay as `rb_can_order`.